packages/xo-web/src/xo-app/proxies/connect-proxy-modal.js

```diff
@@ -0,0 +1,55 @@
+import _ from 'intl'
+import Component from 'base-component'
+import React from 'react'
+import SingleLineRow from 'single-line-row'
+import { Col, Container } from 'grid'
+import { Input as DebounceInput } from 'debounce-input-decorator'
+
+export default class ConnectProxyModal extends Component {
+  state = {
+    authenticationToken: undefined,
+    name: undefined,
+    address: undefined,
+    vmUuid: undefined,
+  }
+  get value() {
+    return this.state
+  }
+
+  render() {
+    const { address, authenticationToken, name, vmUuid } = this.state
+    return (
+      <Container>
+        <SingleLineRow className='mt-1'>
+          <Col size={6}>{_('proxyAuthToken')}</Col>
+          <Col size={6}>
+            <DebounceInput
+              className='form-control'
+              onChange={this.linkState('authenticationToken')}
+              value={authenticationToken}
+            />
+          </Col>
+        </SingleLineRow>
+        <SingleLineRow className='mt-1'>
+          <Col size={6}>{_('name')}</Col>
+          <Col size={6}>
+            <DebounceInput className='form-control' onChange={this.linkState('name')} value={name} />
+          </Col>
+        </SingleLineRow>
+        <SingleLineRow className='mt-1'>
+          <Col size={6}>{_('address')}</Col>
+          <Col size={6}>
+            <DebounceInput className='form-control' onChange={this.linkState('address')} value={address} />
+          </Col>
+        </SingleLineRow>
+        {_('or')}
```

Comment:
It's not necessary to add it, you can add a check before connecting the proxy and an error message for that or an information below.

Reply:
It was to improve the UX.
Without this, the user may think that both fields are optional or required.